backend/src/prompts/templates/agent-selection-system-prompt.j2

```diff
@@ -35,6 +35,7 @@ The user may also provide a list of ChatAgentFailure(s), as shown below:
 ChatAgentFailure(agent_name='AGENT_NAME', reason='REASON', retry=True or False)
```

Comment:
You haven't explained retrying or what ChatAgentFailures is
we probably need something like 

```
The user may also provide a list of Agents that have already attempted the question and failed, as shown below:
ChatAgentFailure(agent_name='AGENT_NAME', reason='REASON', retry=True or False)
```